lib/generated/intl/messages_en.dart

```diff
@@ -1,3 +1,5 @@
+// TODO: remove sdk version selector after migrating to null-safety.
+// @dart=2.10
```

Comment:
Suggest not adding anything into this auto-generated file.

Please help remove them by restart the intl generate process:
`flutter pub global run intl_utils:generate`

If you want to fix the import-non-null-safety warning from the analyzer, please add this comment
```dart
// ignore_for_file: import_of_legacy_library_into_null_safe
```
on the top of the file.

Reply:
solve commit : [22659b7](https://github.com/NEO-TAT/tat_flutter/pull/83/commits/22659b722eac53b1faab42e35b2ba3bf9a341877)